File: Go/1012 - Área.go
package main

import (
	"fmt"
)

func main(){

	var tri, cir, tra, qua, ret float64
	var a, b, c float64

	fmt.Scan(&a, &b, &c)

	tri = (a*c)/2
	cir = (c*c) * 3.14159
	tra = (a+b) * c/2
	qua = b*b
	ret = a*b

	fmt.Printf("TRIANGULO: %.3f\n", tri)
	fmt.Printf("CIRCULO: %.3f\n", cir)
	fmt.Printf("TRAPEZIO: %.3f\n", tra)
	fmt.Printf("QUADRADO: %.3f\n", qua)
	fmt.Printf("RETANGULO: %.3f\n", ret)
}

File: C/1012 - Área.c
#include <stdio.h>

int main(){

	double tri, cir, tra, qua, ret;
	double a, b, c;

	scanf("%lf %lf %lf", &a, &b, &c);

	tri = (a*c)/2;
	cir = (c*c) * 3.14159;
	tra = (a+b) * c/2;
	qua = b*b;
	ret = a*b;

	printf("TRIANGULO: %.3lf\n", tri);
	printf("CIRCULO: %.3lf\n", cir);
	printf("TRAPEZIO: %.3lf\n", tra);
	printf("QUADRADO: %.3lf\n", qua);
	printf("RETANGULO: %.3lf\n", ret);

	return 0;
}